orderer/consensus/etcdraft/chain.go

```diff
@@ -921,37 +925,43 @@ func (c *Chain) ordered(msg *orderer.SubmitRequest) (batches [][]*common.Envelop
 			}
 		}
 
-		if !atomic.CompareAndSwapUint32(&c.leaderTransferFailed, 1, 0) {
-			if c.checkForEvictionNCertRotation(msg.Payload) {
-				go func() {
-					if transferee, err := c.Node.abdicateLeader(c.raftID); err != nil {
-						if err.Error() == LEADERTRANSFERTIMEOUT.Error() && transferee != raft.None {
-							oldTransferee := atomic.LoadUint64(&c.leadershipTransferCandidate)
-							if !atomic.CompareAndSwapUint64(&c.leadershipTransferCandidate, oldTransferee, transferee) {
-								atomic.StoreUint32(&c.leaderTransferFailed, 1)
-							}
-							// sleep for tick time & reattempt
-							timer := c.Node.clock.NewTimer(time.Duration(c.Node.config.ElectionTick) * c.Node.tickInterval)
-							defer timer.Stop()
-
-							select {
-							case <-timer.C():
-							case <-c.Node.chain.doneC:
-							}
-						} else if err.Error() == LEADERTRNSFERCALLEDBYFOLLOWER.Error() {
-							c.logger.Debugf("Leadership transfer completed")
-						} else {
-							atomic.StoreUint32(&c.leaderTransferFailed, 1)
+		if c.checkForEvictionNCertRotation(msg.Payload) {
+
+			if !atomic.CompareAndSwapUint32(&c.leadershipTransferInProgress, 0, 1) {
+				c.logger.Warnf("A reconfiguration transaction is already in progress, ignoring a subsequent transaction")
+				return
```

Comment:
this return statement would return 3 values(batches [][]*common.Envelope, pending bool, err error). do we need to be explicitly return values here?

Reply:
it will return zero values, and then in line 724 it will continue, no?

Reply:
Yes, It will continue. 

Do we need to be explicitly quote the return value like line no. https://github.com/hyperledger/fabric/blob/588ab20a3d00fd88d5b10792b9984a0096d8b4d3/orderer/consensus/etcdraft/chain.go#L964 for code readability

Reply:
I don't think so...